setup-ros.sh

```diff
@@ -31,6 +32,15 @@ for URL in ${ROS_APT_HTTP_REPO_URLS//,/ }; do
     echo "deb ${URL}/ubuntu $(lsb_release -sc) main" >> /etc/apt/sources.list.d/ros-latest.list
 done
 
+case ${ROS_DISTRO} in
+    "kinetic" | "melodic")
```

Comment:
does `noetic` need python-rosdep? Or will it fit into the python3-rosdep bucket

Reply:
As far as I know, it should fall under the `python3-rosdep` bucket.